tests/test_read_range.py

```diff
@@ -62,7 +62,7 @@ def validate_read_range(
         raise AssertionError(error_message)
 
 
-def test_invalid_read_range(
+def manual_test_invalid_read_range(
```

Comment:
looks like we can finally modify this test to be automatic

Reply:
no it's not. we need further modification to automate it